prowler/providers/aws/services/sns/sns_service.py

```diff
@@ -74,6 +75,39 @@ def __list_tags_for_resource__(self):
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def __get_subscriptions__(self):
+        logger.info("SNS - Getting subscriptions...")
+        try:
+            for topic in self.topics:
+                try:
+                    regional_client = self.regional_clients[topic.region]
+                    response = regional_client.list_subscriptions_by_topic(
+                        TopicArn=topic.arn
+                    )
+                    subscriptions: list[Subscription] = [
+                        Subscription(**sub)
+                        for sub in response["Subscriptions"]
+                        if sub["SubscriptionArn"]
+                        != "PendingConfirmation"  # PendingConfirmation is the default name for a subscription that has not been confirmed
+                    ]
+                    topic.subscriptions = subscriptions
+                except Exception as error:
+                    logger.error(
+                        f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+                    )
+        except Exception as error:
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+
+class Subscription(BaseModel):
+    SubscriptionArn: str
+    Owner: str
+    Protocol: str
+    Endpoint: str
+    TopicArn: str
```

Comment:
```suggestion
    arn: str
    owner: str
    protocol: str
    endpoint: str
```

Reply:
Please, use the pattern that we have for the rest of the classes and you don't need the TopicArn since it is already attached to a Topic class.

Reply:
Good suggestion. Fixed!